pinot-core/src/test/java/com/linkedin/pinot/core/segment/index/loader/LoadersTest.java

```diff
@@ -45,6 +47,9 @@
 public class LoadersTest {
   private static final Logger LOGGER = LoggerFactory.getLogger(Loaders.class);
   private static final String AVRO_DATA = "data/test_data-mv.avro";
+  private static final String PADDING_OLD = "data/paddingOld.tar.gz";
```

Comment:
Please add a README file in the data directory that mentions what these files are (preferably the githash with which these files were generated, etc). Regenerating these files once the code is chcked in will produce different results (am I right?) so it will be useful to have a doc with the files.